lib/ProductOpener/Display.pm

```diff
@@ -7248,11 +7248,11 @@ HTML
 
 
 	if ((defined $product_ref->{no_nutrition_data}) and ($product_ref->{no_nutrition_data} eq 'on')) {
-		$html .= "<p>$Lang{no_nutrition_data}{$lang}</p>";
+		$html .= "<div class='panel callout'>$Lang{no_nutrition_data}{$lang}</div>";
+	}
+	else {
```

Comment:
It's still useful to display the nutrition table even if the product has no nutrition data, as it is possible to see the average nutrition values for products in the same category.

Reply:
Aren't average values shown as a percentage?

Reply:
> Aren't average values shown as a percentage?

You can do both https://world.openfoodfacts.org/product/5450168551800/raibi-saveur-grenade-luxlait
![image](https://user-images.githubusercontent.com/87124/59216011-fdd0ac80-8bba-11e9-8e22-ebd5f687c162.png)

Reply:
Even if a product has no nutrition data, we show the average:

![image](https://user-images.githubusercontent.com/8158668/59218244-12637380-8bc0-11e9-9af8-401062bd339a.png)

Reply:
Also, as this information make sense I think it's important to use an id in the HTML code, and not only CSS classes. That way, for example, I can check with my Open Food Facts power user extension if a product hasn't nutrition data on its cover.